README.md

```diff
@@ -73,7 +74,43 @@ TBD
 
 ### Llama.cpp
 
-TBD
+**Step 1**. Build Docker image
+
+```bash
+docker build -t llm-perf-llama-cpp:v0.1 -f Dockerfile.cu121.llama.cpp .
+```
+
+**Step 2**. Fetch quantized GGML binaries and run Llama2 via llama.cpp. Log in to the docker container we created using the comamnd below:
+
+```bash
+PORT=41514
+GGML_BINS=/PATH/TO/GGML_BINS/
+
+docker run                  \
+  -d -P                     \
+  --gpus all                \
+  -h llm-perf               \
+  --name llm-perf-llama.cpp \
+  -p $PORT:22               \
+  -v $GGML_BINS:/ggml_bins  \
+  llm-perf-llama-cpp:v0.1
+
+# Password is: llm_perf
+ssh root@0.0.0.0 -p $PORT
+
+# Inside the container, run the following commands:
+micromamba activate python311
+```
+
+**Step 3.** Run the CLI tool to see the performance numbers:
+
+GGML binary files are like `llama-2-7b.ggmlv3.q4_K_M.bin` or `llama-2-7b.ggmlv3.q4_0.bin`.
+You can download llama-2 7B GGML binary files from [here](https://huggingface.co/TheBloke/Llama-2-7B-GGML/tree/main).
+
+```bash
+cd $LLAMA_CPP_HOME
+./build/bin/main -m /ggml_bins/GGML_BINARY_FILE -p "Please generate a very long story about wizard and technology, at least two thousand words" -n 1920 -ngl 999 --ignore-eos |& tee llama_cpp_result.log
```

Comment:
I followed this instruction but it generates gibberish. Removing `-ngl 999` does make it work and generate sane stuff, but it's incredibly slow

Reply:
well interesting...the `main` from Makefile generates gibberish, but the one from cmake doesn't...

Reply:
Yes, I was referring to the cmake results, somehow the make result doesn't generate anything sane.